docs/content/stable/architecture/docdb-sharding/cluster-balancing.md

```diff
@@ -0,0 +1,234 @@
+- If the node is down for a prolonged time (according to [follower_unavailable_considered_failed_sec](../../../reference/configuration/yb-tserver/#follower-unavailable-considered-failed-sec), which is 15 minutes by default) the cluster balancer starts creating new replicas of the data on the failed node on other nodes in the same region (if any exist).
+- Queries will succeed as soon as tablet leaders move off of the failed node. The data balancing phase does not impact the availability of the database.
+
+#### When the failed node recovers
+
+If or when the failed node recovers, the cluster balancer rebalances tablet leaders onto the recovered node, as well as tablet data (if data had moved off of the node).
+
+### Table and tablet creation and deletion
+
+Operations that create or delete tablets may also trigger cluster balancing. Some examples of operations that may trigger cluster balancing are:
+
+- Creating a table
+- Dropping a table
+- [Tablet splitting](../tablet-splitting/)
+
+## Monitoring cluster balancer progress
+
+While leader rebalancing is fast, data rebalancing can take anywhere from seconds to hours. The major factors influencing how long data rebalancing takes are:
+
+- The size of data on the failed node.
+- The configured network and disk throughput.
+- Cluster balancer configuration flags (described in the [Flags section](#flags)).
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="#yugabyted" class="nav-link active" id="yugabyted-tab" data-bs-toggle="tab" role="tab" aria-controls="yugabyted" aria-selected="true">
+      <img src="/icons/database.svg" alt="Database Icon">
+      yugabyted
+    </a>
+  </li>
+  <li>
+    <a href="#yba" class="nav-link" id="yba-tab" data-bs-toggle="tab" role="tab" aria-controls="yba" aria-selected="false">
+      <img src="/icons/server.svg" alt="Server Icon">
+      YugabyteDB Anywhere
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="yugabyted" class="tab-pane fade show active" role="tabpanel" aria-labelledby="yugabyted-tab">
+
+With yugabyted, you can access the YB-Master UI at <http://localhost:7000> to access two views to monitor the cluster balancer progress.
+
+#### Tablet Servers view
+
+The "Tablet Servers" view contains information about how many peers and leaders are hosted on each tablet server, as well as whether each tablet server is currently data or leader blacklisted.
+
+To access the view, click **Tablet Servers** on your YB-Master UI.
+
+#### Cluster Balancer view
+
+This view is available only in YugabyteDB {{<release "2024.2.6">}} and later, and {{<release "2025.1.1">}} and later.
+
+The Cluster Balancer view displays information about:
+
+- Ongoing cluster balancing tasks (for example, tablet leadership movement, tablets being added or removed).
+- Any warnings during cluster balancing (for example, a placement policy being unsatisfiable because there are no TServers in the requested zone)
```

Comment:
```suggestion
- Any warnings during cluster balancing. For example, if a placement policy cannot be satisfied because there are no TServers in the requested zone).
```